test/e2e/e2e_test.go

```diff
@@ -67,6 +70,9 @@ var _ = Describe("controller", Ordered, func() {
 		By("installing prometheus operator")
 		Expect(utils.InstallPrometheusOperator()).To(Succeed())
 
+		By("installing prometheus")
+		Expect(utils.ApplyPrometheusCRD()).To(Succeed())
```

Comment:
Do we need this? I expect when you install the bundle, you get the CRD directly?

Reply:
Sorry ignore that, this is for the Prometheus object